src/jvm/main/org/jetbrains/kotlinx/lincheck/Result.kt

```diff
@@ -60,18 +60,44 @@ object Cancelled : Result() {
  * Type of result used if the actor invocation fails with the specified in {@link Operation#handleExceptionsAsResult()} exception [tClazz].
  */
 @Suppress("DataClassPrivateConstructor")
-data class ExceptionResult private constructor(val tClazz: Class<out Throwable>, override val wasSuspended: Boolean) : Result() {
+class ExceptionResult private constructor(
+    /**
+     * Exception is stored to print it's stackTrace in case of incorrect results
+     */
+    val throwable: Throwable,
+    /**
+     * Normalized version of exception class
+     */
+    val tClazz: Class<out Throwable>,
```

Comment:
Do you need it? You can compare ExceptionResult's by class names.

Reply:
Are you talking about simple class names? If yes, there might be situations with exceptions with the same names from different packages. If it's about full class name, then do you remember why you initially stored normalized version of class, do we transform exceptions?

Reply:
I mean the full class name. No need to store the `Class` instance.